lib/vc.js

```diff
@@ -43,6 +43,9 @@ const CredentialIssuancePurpose = require('./CredentialIssuancePurpose');
 const defaultDocumentLoader = jsigs.extendContextLoader(
   require('./documentLoader'));
 
+// Constants
+const CREDENTIALS_CONTEXT_V1 = 'https://www.w3.org/2018/credentials/v1';
```

Comment:
We are already depending on `credentials-context` ... can we grab this from there instead?